specification/configuration.md

```diff
@@ -163,6 +163,35 @@ Other requirements:
 - RUM library MUST use the Zipkin v2 JSON span exporter by default
 - RUM library MUST limit the number of sent spans to 100 in a 30 second window per `component` attribute value
 
+#### Serverless 
+
+Current serverless offering is composed of separate metrics (SFx-based) and traces (OTel-based) solutions. 
+It is also assumed that all the components send data directly to Splunk Observability Cloud (direct ingest). Therefore a set of specific configuration properties needs to be defined for the best user experience.
+   
+All serverless solution (wrappers, extensions and other vendor-specific ones) MUST honour the following environment variables: 
+
+| Name                                   | Default value                                                              | Description                                                   |
+| -------------------------------------- | -------------------                                                        | ------------------------------------------------------------- |
+| `SPLUNK_ACCESS_TOKEN`                  | None                                                                       | Access token added to exported data.                          |
+| `SPLUNK_REALM`                         | `us0`                                                                      | Realm configured for the exporter endpoint. [1]               |
+| `SPLUNK_DEBUG`                         | false                                                                      | If set to true, additional debug information will be logged   |
+| `SPLUNK_TRACE_RESPONSE_HEADER_ENABLED` | true                                                                       | Whether Server-Timing header is added to HTTP responses       |
+
+OpenTelemetry-based wrappers MUST honour all configuration properties as defined by the OTel spec.
+Following properties MUST have default values set, as specified:
+- `OTEL_TRACES_EXPORTER` - defaults to `otlp-http`
```

Comment:
There's no such exporter as `otlp-http` - neither in the spec nor in the Java SDK. Perhaps you meant setting this to `otlp` and `OTEL_EXPORTER_OTLP_TRACES_PROTOCOL` to `http/protobuf`?

Reply:
> I can support both propositions. Adding `otlp-http` will certainly simplify life or our customers using direct ingest (also for instrumentations).

If we want to introduce this, we'd have to do it in all instrumentation distros

Reply:
So, can you go back to `otlp` in this PR?

Reply:
Well if we have a wrappers section it can diverge from app configuration. We could also make `otlp-http` optional. Is there no way in spec today to configure between OTLP gRPC vs. HTTP??